docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+[Java SDK Documentation](https://www.javadoc.io/static/io.temporal/temporal-sdk/1.0.0/io/temporal/workflow/Saga.html)
+
+For those using other SDKs, you can implement functions like `addCompensation` and `compensate` to achieve similar behavior.
+
+## Sandbox
+
+| Feature             | Go | Java | TypeScript | Python | .Net | PHP |
+| ------------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [Sandbox](#sandbox) | No | No   | Yes        | Yes    | No   | No  |
+
+Some SDKs support running Workflows inside a sandbox environment.
+
+The Temporal Python SDK, for example, enables you to run Workflow code in a sandbox environment to help prevent non-determinism errors in your application.
+The Temporal Workflow Sandbox for Python is not completely isolated, and some libraries can internally mutate state, which can result in breaking determinism.
+
+<!-- V8 Isolates TypeScript -->
+
+By default, Workflows run in a sandbox environment.
+If a Workflow Execution performs a non-deterministic event, an exception is thrown, which results in failing the Workflow Task.
+The Workflow will not progress until the code is fixed.
+
+## Scheduling
+
+| Feature                 | Go  | Java | TypeScript | Python | .Net | PHP |
+| ----------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Cron](#cron)           | Yes | Yes  | Yes        | Yes    | No   | Yes |
+| [Schedules](#schedules) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+### Cron
+
+Cron is a scheduling system that allows you to schedule Workflow Executions to run at specific times.
+It is recommened to use [Schedules](#schedules) instead of Cron Jobs.
+
+### Schedules
+
+A Schedule contains instructions for starting a [Workflow Execution](/workflows#workflow-execution) at specific times.
+Schedules provide a more flexible and user-friendly approach than [Temporal Cron Jobs](/workflows#temporal-cron-job).
+
+## SDK Metrics
+
+| Feature                     | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [SDK Metrics](#sdk-metrics) | Yes | Yes  | Yes        | Yes    | Yes  | No  |
+
+SDK Metrics track various aspects of Worker performance, such as Task Queue, Namespace, poller type, Worker type, Activity type, Workflow type, and operation.
+
+For more information on metrics see, [Temporal SDK metrics reference](/references/sdk-metrics#).
+
+## Search Attributes
+
+| Feature                                 | Go  | Java | TypeScript | Python | .Net | PHP |
+| --------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Search Attributes](#search-attributes) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+A Search Attribute is an indexed field used in a [List Filter](/visibility#list-filter) to filter a list of [Workflow Executions](/workflows#workflow-execution) that have the Search Attribute in their metadata.
+
+## Sessions
```

Comment:
I don't think sessions are worth calling out as a "feature". I support documentation around worker-specific task queues (but not as a language comparison below), but sessions should not be discussed in a global way